src/osmviz/manager.py

```diff
@@ -190,11 +190,27 @@ def __init__(self, **kwargs):
                     should be fully qualified, including the protocol.
                     Default "http://tile.openstreetmap.org"
 
+        url - Full URL template from which to retrieve OSM tiles. This should
+                    be fully qualified, including the protocol, and should
+                    contain placeholders for zoom ('{z}'), coordinate x and y
+                    ('{x}' and '{y}'), and optionally scale ('{s}') for hi-dpi
+                    tile retrieval.
+                    Note: when specified, the server parameter is ignored.
+                    Default: server appended with "/{z}/{x}/{y}.png"
+
+        scale - Scale to use for hi-dpi tiles. Note that both url and scale
```

Comment:
Not everyone seems to agree on how to call these tiles, but in the end it doesn't matter that much I guess. I'll go ahead and change those occurrences.

Counter examples:

Example from OpenLayers uses both "high DPI" and "hiDPI": http://openlayers.org/en/latest/examples/wmts-hidpi.html
Mapbox docs talk about "high DPI": https://www.mapbox.com/developers/api/v3/maps/#format
They (and others) also use "Retina" btw: https://www.mapbox.com/api-documentation/#retina